### DISCUSSION TITLE
Weblate dockerized deployment - Scale out Celery

### DISCUSSION THREAD

**Discussion by csuriano23** [WeblateOrg/weblate#11046] [Q&A] [answered]
I've read [in the docs](https://docs.weblate.org/en/latest/admin/install/docker.html#envvar-WEBLATE_SERVICE) that the suggested way in order to horizontally scale weblate processes is to define a service for each component, eg. celery workers.
Would it be possible to deploy the celery master worker as a standalone container/process? If yes, are there any examples and caveats on how to do that?

**Comment by nijel** [comment, marked as answer]
What do you call a “celery master worker”? The Celery workers are described it the documentation (which you've linked), and there is no master among them. All workers are equal, and they only differ in configured queue to process. The only specific one is `celery-beat` which does not process any tasks, it just triggers scheduled background tasks which will be then executed by the other workers.

**Reply by csuriano23** [reply]
Thank you @nijel , so from what I understand there is no need to start other celery processes apart from the ones described in docs. I thought it would be needed for additional processes like a Flower dashboard, but I guess that this could be started apart from the standard weblate processes

**Reply by nijel** [reply]
If you want Flower dashboard, you need to executed that standalone. If you just want to monitor length of Celery queues, you can do that within Weblate.
